src/commands/BootstrapCommand.js

```diff
@@ -92,18 +92,40 @@ export default class BootstrapCommand extends Command {
   bootstrapPackages(callback) {
     this.logger.info("", `Bootstrapping ${this.filteredPackages.length} packages`);
 
-    async.series([
-      // preinstall bootstrapped packages
-      (cb) => this.preinstallPackages(cb),
-      // install external dependencies
-      (cb) => this.installExternalDependencies(cb),
-      // symlink packages and their binaries
-      (cb) => this.symlinkPackages(cb),
-      // postinstall bootstrapped packages
-      (cb) => this.postinstallPackages(cb),
-      // prepublish bootstrapped packages
-      (cb) => this.prepublishPackages(cb)
-    ], callback);
+    const { yarnWorkspaces } = this.options;
+
+    if (yarnWorkspaces && (yarnWorkspaces.indexOf("bootstrap") >= 0)) {
+      this.installRootPackageOnly(callback);
```

Comment:
No symlinking happens in this case? Or is that handled by Yarn now? (I dug through https://github.com/yarnpkg/yarn/issues/3294 but my eyes glazed over before I could figure it out, sorry)